src/evaluations/data/evaluation_configs.py

```diff
@@ -80,7 +80,7 @@
 
 
 # Document the evaluation configurations.
-def _smoke_test(num_runs=NUM_RUNS_VALUE):
+def _smoke_test(num_runs=NUM_RUNS_VALUE, universe_size=200000):
```

Comment:
2e5 is in several places. Does it make sense to create a global variable: DEFAULT_UNIVERSE_SIZE?

Reply:
Done.